src/contracts/periphery/OperatorFactory.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity >=0.8.0;
+
+import "./Operator.sol";
+import "@openzeppelin/contracts/utils/Create2.sol";
+
+contract OperatorFactory {
+    function createOperator() external returns (Operator) {
+        return Operator(
+            Create2.deploy(
```

Comment:
You don't need to use create2 library anymore. Solidity supports native create2 by just using `{salt: _salt}`. Example: https://solidity-by-example.org/new-contract/